site/content/entry/accurate-audio-emulation.md

```diff
@@ -0,0 +1,199 @@
+The game would transition from state to state, and mysteriously whenever the game softlocked, it simply wasn't moving onto the next state.
+As cool as it is to what causes the softlock, it doesn't answer the big question of *why* doesn't the game transition to the next state like it should.
+After spending more time than anyone could have asked, eventually he burned out and moved on to develop other amazing projects for citra such as multiplayer network support, leaving us without any more clues to why the game freezes.
+
+{support convo from fates? tomodachi life voices?}
+
+## Then Is the Problem Everyone's Best Guess: AAC Audio?
+
+Well-informed users have pointed out for years that the specific sounds that are not playing in Citra when emulating Pokémon X all have a very suspicious thing in common: all of them are stored in AAC audio format.
+AAC (or [Advanced Audio Coding](https://en.wikipedia.org/wiki/Advanced_Audio_Coding)) is a standard audio format that typically has better sounding audio than mp3 at the same bitrate.
+While we appreciate the detective work, there was one glaring problem with deciding that we just needed to add AAC audio support.
+How *does* one add AAC audio support to Citra's HLE audio?
+The answer was talked about earlier, in order to add a new feature in HLE audio, one needs to reverse engineer the audio code that the games use, and then find out exactly how the audio code processes AAC audio; where the data is passed through, where each decoding option is stored, and all the way down what every bit of data written to the audio pipe semantically means.
+To make matters worse, there's no guarentee that this will fix any other games with similar symptoms.
+After all, a game can upload any code it wants to the audio chip, meaning even if they both used AAC audio, they could potentially use different decoding options, causing the HLE AAC decoder to work for one game and not the other.
+And worst of all, it's possible that everyone is wrong, and X/Y are freezing because of a completely unrelated issue!
+
+Faced with this dilemma, [wwylele](https://github.com/wwylele) designed a test to show that audio is likely the cause of the softlock.
+He knew from the get go that the HLE audio code was based of reverse engineering the most common audio code that games use, the home menu, and decided to make a custom rom hack for Pokémon X that replaced it's audio code with the home menu audio.
+Upon launching the game, everything seemed fine at first, but soon enough the familiar lack of background music kicked in, and Pokémon X started to behave exactly like it does in Citra.
+One short level up later, and the game froze, just like in Citra!
+We double checked the results by recreating this on different copies of Pokémon X and different 3dses, and it all went the exact same way.
+Audio issues is very likely the cause, but what should the fix be?
```

Comment:
"Audio issues were very likely the cause". Both rolls off the tongue better, and these issues are solved(ish) now. Or maybe "The cause is very likely audio issues, but"